config/routes.rb

```diff
@@ -68,6 +68,7 @@
         post 'bulk_import_students'
         get 'evaluation_criteria'
         post 'attach_images'
+        get 'clone'
```

Comment:
The cloning route should not be a GET. This is not an idempotent operation, and more importantly, it's not a _safe_ operation to use a GET method on.